lib/smart_app_launch/token_payload_validation.rb

```diff
@@ -132,5 +132,31 @@ def validate_fhir_context(fhir_context)
         assert id.match?(FHIR_ID_REGEX), "`#{id}` is not a valid FHIR id"
       end
     end
+
+    def validate_fhir_context_stu2_2(fhir_context)
+      return if fhir_context.nil?
+
+      assert fhir_context.is_a?(Array), "`fhirContext` field is a #{fhir_context.class.name}, but should be an Array"
+
+      fhir_context.each do |reference|
+        assert reference.is_a?(Hash), "`#{reference.inspect}` is not a Hash"
+      end
+
+      fhir_context.each do |reference_obj|
+        reference = reference_obj['reference']
+
+        assert reference.present?,
```

Comment:
These are not the requirements described in the IG: http://hl7.org/fhir/smart-app-launch/scopes-and-launch-context.html#fhircontext-exp

Reply:
Yeah you are right, I didn't see the section below explaining all of the new fields that could be included and originally just thought the format of references was updated. Fixed to include all new requirements described in that section